### PR TITLE
Add Gledopto GL-C-008P firmware preinstalled on recent models

### PR DESCRIPTION
**Pull request by MartB**
This firmware already runs on my GL-C-008P, the file is from Gledopto.
This silences the the "newer than available" warning.

### CODE REVIEW

File: index.json [hunk 1]
```diff
@@ -1777,5 +1777,14 @@
         "sha512": "b136d2656ea1197ae84f5e9c2244a9d9697bccab2c448324d9176bac791aff0161afc73c8a1574bbd51c0ea2e318840ca58a0a99da32669d3e5fc776bdf3473a",
         "url": "https://github.com/Koenkk/zigbee-OTA/raw/master/images/Sonoff/zbmini-l_v1.1.1.ota",
         "path": "images/Sonoff/zbmini-l_v1.1.1.ota"
+    },
+    {
+        "fileVersion": 18,
+        "fileSize": 291770,
+        "manufacturerCode": 4687,
```
Comment (Koenkk) [inline index.json:1784]: Please add the modelid and remove the existing entry: https://github.com/Koenkk/zigbee-OTA/blob/9c2cee96b280d33c0123c08e3dd599ad69094e32/index.json#L922

Comment (MartB) [reply index.json]: Whoops, i forgot to add the modelId as the second argument to add.js!
Fixed!

**Comment by Koenkk** [comment]
Thanks!

**Comment by MartB** [comment]
> Reverted this because of [Koenkk/zigbee2mqtt#12741](https://github.com/Koenkk/zigbee2mqtt/issues/12741)

Whew thats odd, seems like they used a different chip for the older variants :s

**Pull request merged**